modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -248,8 +537,9 @@ The following table describes each update:
 \* Before, the types were too generic and not very useful. They now better reflect the values they
 represent.
 
-The codemod will handle _almost all_ of these changes with a few possible exceptions noted in
-[Manual Updates](#manual-updates) below.
+The codemod will handle _almost all of_ of these changes with a The v5 codemod handles aew possible
+exceptions not. for you. That said, you'll want to review your UI to ensure everything was updated
```

Comment:
I think there was a copy-and-paste accident here 😄

That, or one of your kids got ahold of your keyboard.

Reply:
Yep. This is even the wrong section. Sheesh. Good catch.